py3status/modules/dropboxd_status.py

```diff
@@ -30,6 +32,18 @@
 Requires:
     dropbox-cli: command line interface for dropbox
 
+Notes:
+Some distributions offer an option to install dropbox-cli. If you don't see
+one for your distribution, then you need to download CLI Python script from
+Dropbox, rename it to dropbox-cli and make dropbox-cli executable and available
+in your PATH.
+See https://www.dropbox.com/help/desktop-web/linux-commands#commands for
+the python script.
+
+dropbox-cli is available on:
+    - Arch Linux's AUR/dropbox-cli
+    - Gentoo's net-misc/dropbox-cli
```

Comment:
I think adding...
```
dropbox-cli is available on:
    - Arch Linux's AUR/dropbox-cli
    - Gentoo's net-misc/dropbox-cli
```
... and not indenting may mess up the `readthedocs` documentation. Can you use this diff?
```diff
diff --git a/py3status/modules/dropboxd_status.py b/py3status/modules/dropboxd_status.py
index b607354..e1b57dc 100644
--- a/py3status/modules/dropboxd_status.py
+++ b/py3status/modules/dropboxd_status.py
@@ -5,9 +5,12 @@ Display status of Dropbox daemon.
 Configuration parameters:
     cache_timeout: refresh interval for this module (default 10)
     format: display format for this module (default "Dropbox: {status}")
-    status_busy: text for placeholder {status} when Dropbox is busy (default None)
-    status_off: text for placeholder {status} when Dropbox isn't running (default "isn't running")
-    status_on: text for placeholder {status} when Dropbox is up to date (default "Up to date")
+    status_busy: text for placeholder {status} when Dropbox is busy
+        (default None)
+    status_off: text for placeholder {status} when Dropbox isn't running
+        (default "isn't running")
+    status_on: text for placeholder {status} when Dropbox is up to date
+        (default "Up to date")
 
 Value for `status_off` if not set:
     - Dropbox isn't running!
@@ -33,16 +36,10 @@ Requires:
     dropbox-cli: command line interface for dropbox
 
 Notes:
-Some distributions offer an option to install dropbox-cli. If you don't see
-one for your distribution, then you need to download CLI Python script from
-Dropbox, rename it to dropbox-cli and make dropbox-cli executable and available
-in your PATH.
-See https://www.dropbox.com/help/desktop-web/linux-commands#commands for
-the python script.
-
-dropbox-cli is available on:
-    - Arch Linux's AUR/dropbox-cli
-    - Gentoo's net-misc/dropbox-cli
+    Some distributions offer an option to install dropbox-cli. If you don't see
+    one from your distribution, then you need to download CLI Python script,
+    https://www.dropbox.com/help/desktop-web/linux-commands#commands, rename
+    it to `dropbox-cli`, make the script executable and available in your PATH.
 
 @author Tjaart van der Walt (github:tjaartvdwalt)
 @license BSD
```